overrides/config/betterquesting/DefaultQuests.json

```diff
@@ -21730,7 +21730,7 @@
       "properties:10": {
         "betterquesting:10": {
           "autoclaim:1": 0,
-          "desc:8": "An otherworldly material exclusive to Microverses, §6Naquadah§r is an important material in the late game.\n\nUnlike most materials at this point, Naquadah can be smelted in a standard furnace.\n\nIt can be alloyed and enriched to various forms, and is the eventual source of §6Neutronium§r in a §3Mark Three Fusion Reactor§r.",
+          "desc:8": "An otherworldly material exclusive to Microverses, §6Naquadah§r is an important material in the late game.\n\nYou\u0027ll need to upgrade your §3Electric Blast Furnace§r with a set of §6HSS-G§r coils to smelt this material.\n\nIt can be alloyed and enriched to various forms, and is the eventual source of §6Neutronium§r in a §3Mark Three Fusion Reactor§r.",
```

Comment:
This mention might be redundant since HSS-G coils are a prerequisite, and your change to the coils quest mentions it's needed for Naquadah. Should be sufficient to just remove the old sentence about smelting in a furnace.

Reply:
I was thinking about that as well, figured I'd get some feedback first.